modules/tinymce/src/themes/silver/main/ts/ui/context/ContextFormApi.ts

```diff
@@ -7,17 +7,22 @@ import {
 } from '@ephox/alloy';
 import { InlineContent } from '@ephox/bridge';
 import { Singleton } from '@ephox/katamari';
+import { Focus } from '@ephox/sugar';
 
 import * as ContextToolbarFocus from './ContextToolbarFocus';
 import { backSlideEvent } from './ContextUi';
 
-export const getFormApi = <T>(input: AlloyComponent): InlineContent.ContextFormInstanceApi<T> => {
+export const getFormApi = <T>(input: AlloyComponent, focusfallbackComp?: AlloyComponent): InlineContent.ContextFormInstanceApi<T> => {
   const valueState = Singleton.value<T>();
 
   return ({
     setInputEnabled: (state: boolean) => {
       if (!state) {
-        ContextToolbarFocus.focusParent(input);
+        if (focusfallbackComp) {
+          Focus.focus(focusfallbackComp.element);
+        } else {
+          ContextToolbarFocus.focusParent(input);
```

Comment:
I'm not sure we need this call or the focusParent anymore since there is no tabindex=-1 on any parent of the input element if you add the `ignore` option and also the `focusParent` was a bit hacky anyway so lets kill it.

Reply:
done, I tried to remove entirely the functionality but is also used in: `modules/tinymce/src/themes/silver/main/ts/ui/context/ContextFormGroup.ts`